plugin/trino-tpcds/src/test/java/io/trino/plugin/tpcds/TestTpcds.java

```diff
@@ -86,6 +86,12 @@ public void testShowTables()
         assertQueryFails("SHOW TABLES FROM sf0", "line 1:1: Schema 'sf0' does not exist");
     }
 
+    @Test
+    public void testDate()
+    {
+        assertQuery("SELECT d_date FROM date_dim WHERE d_date_id = 'AAAAAAAAOKJNECAA'", "SELECT DATE '1900-01-02'");
```

Comment:
was this test failing without the fix? Worth a comment (and maybe rename) to explain what exactly are we testing here.

Reply:
I does fail without the fix. The tests are run in `America/Bahia_Banderas` time zone (`-6`). Before the change the value returned used to be `1900-01-01` while the correct value is `1900-01-02` (verified by manually checking the value TPC-DS generator returns as a string)